kolibri/core/urls.py

```diff
@@ -33,15 +33,13 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django.conf import settings
-from django.conf.urls import url
+from django.conf.urls.static import static
 from kolibri.plugins.registry import get_urls as plugin_urls
 
 app_name = 'kolibri'
 
 
 urlpatterns = plugin_urls()
 
-urlpatterns += [
-    url(r'^' + settings.STORAGE_URL[1:-1] + '(?P<path>.*)$', 'django.views.static.serve',
-        {'document_root': settings.STORAGE_ROOT})
-]
+urlpatterns += static(settings.CONTENT_STORAGE_URL, document_root=settings.CONTENT_STORAGE_DIR)
+urlpatterns += static(settings.CONTENT_DATABASE_URL, document_root=settings.CONTENT_DATABASE_DIR)
```

Comment:
Even though we're not currently downloading content DBs across devices, we will be soon!